templates/controllers/tab/settings/navigationMenus/form/navigationMenuSettingsForm.tpl

```diff
@@ -0,0 +1,32 @@
+{**
+ * controllers/tab/settings/navigationMenus/form/NavigationMenuSettingsForm.tpl
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * NavigationMenus settings form.
+ *
+ *}
+
+{* Help Link *}
+{help file="settings.md" section="website" class="pkp_help_tab"}
```

Comment:
We'll probably want to add a section to the help documentation for navigation menus.